project/settings/defaults.py

```diff
@@ -190,7 +204,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'America/New_York'
```

Comment:
This is the [default "display" timezone](https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-TIME_ZONE). Everything is always stored in UTC in the database. Django just uses this setting to change times before they're shown to the user unless the user has a custom timezone set. We're pulling one over from the API for them if they login through oauth. I will double check that we're actually using that to show the user times in their local tz.